sdk/crystal/src/kadalu_storage_manager.cr

```diff
@@ -45,8 +45,8 @@ module StorageManager
       Pool.new(self, name)
     end
 
-    def list_config_snapshots
-      ConfigSnapshot.list(self)
+    def list_config_snapshots(snap_name)
```

Comment:
This change not required.

Reply:
ok. Added new function for getting single snapshot. `get_config_snapshot`